l10n/de/browser/installer/nsisstrings.properties

```diff
@@ -26,13 +26,22 @@ STUB_CLEANUP_PAVEOVER_HEADER=$BrandShortName ist bereits installiert.\nLass uns
 STUB_CLEANUP_REINSTALL_HEADER=$BrandShortName war schon einmal installert.\nLass uns ihn neu installieren.
 STUB_CLEANUP_PAVEOVER_BUTTON=&Aktualisieren
 STUB_CLEANUP_REINSTALL_BUTTON=&Neu installieren
-STUB_CLEANUP_CHECKBOX_LABEL=Einstellungen &zurücksetzen und alte Add-ons entfernen für beste Leistung
+STUB_CLEANUP_CHECKBOX_LABEL=Einstellungen &zurücksetzen für beste Leistung
 
 STUB_INSTALLING_LABEL2=Wird jetzt installiert…
-STUB_BLURB_FIRST1=Der am schnellsten reagierende $BrandShortName bislang
-STUB_BLURB_SECOND1=Schnelleres Laden von Seiten und Wechseln von Tabs
-STUB_BLURB_THIRD1=Privates Surfen leicht gemacht
-STUB_BLURB_FOOTER2=Für die Menschen, nicht für den Profit.
+STUB_BLURB_FOOTER2="Surfen ohne Kompromisse."
+STUB_BLURB1_FIRST1="Danke, dass du $BrandShortName gewählt hast!"
+STUB_BLURB1_SECOND1="Sicher durchs Internet"
+STUB_BLURB1_THIRD1="Schneller am Ziel"
+STUB_BLURB1_FOURTH1="Auch zum Mitnehmen"
+STUB_BLURB1_FIFTH1="Videos offline nutzen"
+STUB_BLURB2_FIRST1=" "
+STUB_BLURB2_SECOND1="$BrandShortName Anti-Tracking schützt deine persönlichen Daten vor der Weitergabe an Dritte."
+STUB_BLURB2_THIRD1="Die $BrandShortName Schnellsuche zeigt dir sofort die direkten Top-Links für deine Suche."
+STUB_BLURB2_FOURTH1="Hol dir $BrandShortName als mobile App und verbinde $BrandShortName auf deinen Geräten."
+STUB_BLURB2_FIFTH1="Lade YouTube Videos einfach auf Computer oder Handy herunter."
+MINIINSTALLER_32_ERROR_SUPPORT_PAGE=https://cliqz.com/installer-help
+MINIINSTALLER_64_ERROR_SUPPORT_PAGE=https://cliqz.com/installer-help-64
```

Comment:
@alver-cliqz will this page be different? currently a 404

Reply:
It will be in next version of website - in development right now